### PR TITLE
Generate ci-kubernetes-e2enode-cosstable1-* tests

### PR DESCRIPTION
**Pull request by yguo0905**
cosstable1 is m60 now. Add the node e2e tests for m60 (Docker 1.13) validation.

/assign @krzyzacy 
/cc @abgworrall

### CODE REVIEW

File: testgrid/config/config.yaml [hunk 1]
```diff
@@ -2980,85 +2998,103 @@ dashboards:
...
+  - name: e2enode-cosstable1-k8sstable2-serial
+    test_group_name: ci-kubernetes-e2enode-cosstable1-k8sstable2-serial
+  - name: e2enode-cosstable1-k8sstable3-default
+    test_group_name: ci-kubernetes-e2enode-cosstable1-k8sstable3-default
+  - name: e2enode-cosstable1-k8sstable3-serial
+    test_group_name: ci-kubernetes-e2enode-cosstable1-k8sstable3-serial
+  - name: e2e-gce-cosdev-k8sdev-default
     test_group_name: ci-kubernetes-e2e-gce-cosdev-k8sdev-default
-  - name: gce-cosdev-k8sdev-slow
+  - name: e2e-gce-cosdev-k8sdev-slow
```
Comment (krzyzacy) [inline testgrid/config/config.yaml]: nit: I think you can save all the e2e- in the tab name.. (less crowded)

Comment (yguo0905) [inline testgrid/config/config.yaml]: Wanted to make it clear that there are both nodee2e and e2e. But it shouldn't be an issue if we have (non-top-level) sub-tabs in the future.

**Comment by yguo0905** [comment]
Good to merge?

**Comment by krzyzacy** [comment]
forgot to push the button :-)

**Pull request merged**